### DISCUSSION TITLE
Help out with React Native releases!

### DISCUSSION THREAD

**Discussion by lunaleaps** [reactwg/react-native-releases#11] [Announcements]
### Update (01/29): Going to lock this for now for those who volunteered

---
We have been updating our release process and part of that work is defining responsibilities of Meta in supporting a release ranging from time/effort.

[Please read over our role definitions here](https://github.com/facebook/react-native/wiki/Release-Roles-and-Responsibilities).

If you are interested in helping as a release supporter, please comment down below with preferred method of contact and we will reach out!

If you are not able to formally commit to any roles, any help trying out [release candidates](https://github.com/facebook/react-native/releases) and [reporting issues](https://github.com/facebook/react-native/issues/new/choose) and or jumping into [debugging current blockers](https://github.com/reactwg/react-native-releases/discussions/categories/releases) is always helpful! Thank you! 

---

Small update (26th Jan): we are still ironing out a few details, expect us to reach out over the next 7 days!

**Comment by Saadnajmi** [comment]
Hello, I am interested in being a release supporter! I am on the contributors discord already, so there would be preferred =)

**Comment by Megatron4537** [comment]
Hello, I am interested. I have been working with React-Native for 3+ years now and contributed to a few packages. Cheers

Contact Info: nicolashortaadam@gmail.com

**Reply by kelset** [reply]
hey @Megatron4537 - one small question, do you want to help as a supporter or a release tester? :)

**Reply by Megatron4537** [reply]
Hey @kelset, I would say as a release tester for now :)

**Comment by fabOnReact** [comment]
Thanks a lot. I'm available in helping out as Release Supporter. I will start giving a look at the issues https://github.com/facebook/react-native/labels?q=release and helping with the [release testing](https://github.com/facebook/react-native/wiki/Release-Testing). 

I'm available on discord and my email is fabrizio.bertoglio@gmail.com

**Comment by kmsbernard** [comment]
Hi, I'm interested in helping as Release Supporter. You can contact me by email at kmsbernard@gmail.com
Thanks a lot!

**Comment by ahmetbicer** [comment]
Hi, I'm interested in helping as a Release Tester. I can test new releases in 2 production apps. You can contact me via email at ahmtbicertr@gmail.com

Thanks!

**Comment by numandev1** [comment]
Hello, I am interested in being a release supporter. I can enable you to ship the stable fabric and turbomodule  Architecture. you can see my comments https://github.com/facebook/react-native-website/pull/2879

Contact Info: Muhammadnuman70@gmail.com

[https://numan.dev](https://numan.dev)

**Comment by diegolmello** [comment]
Hey, everybody! I want to help as Release Tester.
I'm a maintainer of Rocket.Chat and we're open source https://github.com/RocketChat/Rocket.Chat.ReactNative/
You can reach me at diegolmello@gmail.com or @diegolmello on Twitter.
Thanks! :)

**Comment by doranteseduardo** [comment]
Hello!
I am interested on helping as Release Supporter, I'm already on the contributors discord server so you can write me on there.
Thanks in advance!

**Comment by ken0nek** [comment]
Hello, I am interested in being a Release Supporter!

My email address is `ken.for.develop@gmail.com`

**Comment by gabrieldonadel** [comment]
Hey, I'm interested in helping as a release supporter as well, my email is donadeldev@gmail.com

**Comment by caner-karaman** [comment]
Hello, I'm interested in helping as a release supporter
email: karamancaner1995@gmail.com
